boards/xtensa/m5stack_core2/m5stack_core2.dts

```diff
@@ -0,0 +1,237 @@
+/*
+ * Copyright (c) 2023 Martin Kiepfer <mrmarteng@teleschirm.org>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+/dts-v1/;
+
+#include <espressif/esp32/esp32_common.dtsi>
```

Comment:
Please use the `esp32/esp32_wrover_e_n16r8.dtsi` instead of `esp32_common.dtsi`. There are flash and psram attributes included.

Reply:
M5Stack-Core2 is featuring `esp32_d0wd_v3`. The implementation on the m5stack hardware is very near to `esp32_wrover_e_n16r8`, but I found it more consistent to keep sram and flash definition separately within board layer as the wrover module is not physically included. That was the reason I didn't base on the n16r8 intentionally.